articles/django-app-using-postgresql-database/index.md

```diff
@@ -0,0 +1,211 @@
+# Creating a Django Hello World App Using PostgreSQL Database
+> Django is a high-level Python Web framework that encourages rapid development and clean, pragmatic design. Built by experienced developers, it takes care of much of the hassle of Web development, so you can focus on writing your app without needing to reinvent the wheel. It’s free and open source <cite>[Django](https://www.djangoproject.com)<cite>
+
+> PostgreSQL is a powerful, open-source object-relational database system that uses and extends the SQL language combined with many features that safely store and scale the most complicated data workloads. <cite>[PostgreSQL](https://www.postgresql.org/about/) <cite>
+
+### Prerequisites
+To follow this tutorial you need to have
+ - [Python3](https://www.python.org/downloads/)
+ - [PostgreSQL](https://www.postgresql.org/download/windows/)
+ - [virtualenv](https://pypi.org/project/virtualenv/)
+
+### Step 1 - Create a database and database user we will be creating our database through the command line.
```

Comment:
Keep the headings short. 

`we will be creating our database through the command line.`

This line can be added inside the section.

Reply:
This is not done yet.